llvm/lib/CodeGen/GlobalISel/CallLowering.cpp

```diff
@@ -798,15 +857,19 @@ bool CallLowering::handleAssignments(ValueHandler &Handler,
         // store. We may need to adjust the offset for big endian targets.
         LLT MemTy = Handler.getStackValueStoreType(DL, VA, Flags);
 
-        MachinePointerInfo MPO;
+        MachinePointerInfo MPO(ArgReg);
```

Comment:
It is getStackAddress that should initialize MPO.

Reply:
Should getStackAddress be used in every case or just for indirectPassing?  Originally this was just `MachinePointerInfo MPO;`.

Reply:
Oh I understand the register that comes from `getStackAddress`. I changed this. I will also run the regression tests, because it is in code that is not just active for indirect parameter passing.

Reply:
I will reset this to it's original state.